apps/nextjs/src/app/_components/_landing/Hero.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
```

Comment:
We should look to make all of these server components! Although for now, it makes sense to just throw in `use client;` but over time we should definitely remove them and make them server components out of the box.

Reasoning: This is on the landing page - we want fast loads, good SEO crawls, and low TTFP (time to first paint).